numpy/random/mtrand/mtrand.pyx

```diff
@@ -3426,7 +3426,8 @@ cdef class RandomState:
         Returns
         -------
         samples : ndarray or scalar
-                  where the values are all integers in  [0, n].
+            Samples from logistic distribution, shaped according to
+            `size`. Otherwise, a single value is returned.
```

Comment:
is the "Otherwise..." really needed? I feel that given the explanation on the `size` parameter, this is more confusing than helpful. Also, `size=None` does not necessarily mean a single return if the other parameters are arrays, right? Again, I'd make that change in `size`, not here.